DEV.md

```diff
@@ -13,8 +13,9 @@
 
 - Requires that you are listed as a maintainer for this npm module
 
-* Commit and push changes
-* Login: `npm login`
 * Make sure you can run build successfully: `npm run build`
 * Update version: `npm version patch`
```

Comment:
Should this indicate other version types you could create? `npm version {major|minor|patch}`?